.vscode/settings.json

```diff
@@ -1,24 +1,28 @@
 {
-    "tabnine.experimentalAutoImports": true,
-    "go.lintTool": "golangci-lint",
-    "go.formatTool": "gofumports",
-    "go.testOnSave": true,
-    "go.useLanguageServer": true,
-    "[go.mod]": {
-        "editor.formatOnSave": true,
-        "editor.codeActionsOnSave": {
-          "source.organizeImports": true
-        }
-    },
-    "[go]": {
-        "editor.snippetSuggestions": "none",
-        "editor.formatOnSave": true,
-        "editor.codeActionsOnSave": {
-          "source.organizeImports": true
-        }
-    },
-      "gopls": {
+  "tabnine.experimentalAutoImports": true,
+  "go.lintTool": "golangci-lint",
+  "go.formatTool": "gofumports",
+  "go.testOnSave": true,
```

Comment:
I'm personally used to working with auto-save and just noticed the following:
![image](https://user-images.githubusercontent.com/34196718/162316400-9103a54d-232a-43d5-93e5-87cc44356843.png)

Can we let everyone decide if they would like this enabled?

Reply:
Yes, wanna make a followup PR?

Reply:
On it